src/tasks/WasmAppBuilder/PInvokeTableGenerator.cs

```diff
@@ -326,7 +326,7 @@ private void EmitNativeToInterp(StreamWriter w, List<PInvokeCallback> callbacks)
             sb.Append($" (*WasmInterpEntrySig_{cb_index}) (");
             int pindex = 0;
             if (method.ReturnType.Name != "Void") {
-                sb.Append("int");
+                sb.Append("int*");
```

Comment:
@radical @BrzVlad Is this right? does the interp entry function take the address of the return arg?

We didn't hit this before?

Reply:
Pushed this out as a standalone PR #59968